scripts/ec2_stage_arm.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+# Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+#  with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+DRYRUN=true
+
+AWS_PROFILE=""
+AWS_REGION="us-east-1"
+AMI_ID="ami-053b2a8c2f3e87928" #amzn2-ami-hvm-2.0.20181020.0-aarch64-gp2
```

Comment:
So this is the only AMI_ID? Is there a possibility that this will be updated later or there will be more amis needed?

Reply:
Ideally we won't need to update this AMI id in future. 

We do plan to move the build process to codebuild when they add the support for Arm. This script would then get redundant.